app-misc/calamares-config-livecd/calamares-config-livecd-3.3.14.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="8"
+
+MY_PN="calamares"
+MY_P="${MY_PN}-${PV}"
+
+inherit desktop
+
+DESCRIPTION="Gentoo Linux ${MY_PN} installer config"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI="mirror://gentoo/gentoo-artwork-livecd-2007.0.tar.bz2
+		mirror://gentoo/gentoo-artwork-0.2.tar.bz2"
+
+S="${WORKDIR}"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+RDEPEND="app-admin/calamares"
+
+src_prepare() {
+    default
+
+    local baselayout_version
+    baselayout_version="$(best_version sys-apps/baselayout | sed -E 's:.*/[^-]+-::')"
```

Comment:
This looks quite fragile.  How about something like this:
```
baselayout_version=$(best_version sys-apps/baselayout)
baselayout_version=${baselayout_version#*layout-}
```

Reply:
> This looks quite fragile. How about something like this:
> 
> ```
> baselayout_version=$(best_version sys-apps/baselayout)
> baselayout_version=${baselayout_version#*layout-}
> ```

definitely much better